itest/list_on_test.go

```diff
@@ -5,426 +5,430 @@ package itest
+	//},
+	//{
+	//	Name: "revoked uncooperative close retribution zero value " +
+	//		"remote output",
+	//	TestFunc: testRevokedCloseRetributionZeroValueRemoteOutput,
+	//},
+	//{
+	//	Name:     "revoked uncooperative close retribution remote hodl",
+	//	TestFunc: testRevokedCloseRetributionRemoteHodl,
+	//},
+	//{
+	//	Name:     "single-hop send to route",
+	//	TestFunc: testSingleHopSendToRoute,
+	//},
+	//{
+	//	Name:     "multi-hop send to route",
+	//	TestFunc: testMultiHopSendToRoute,
+	//},
+	//{
+	//	Name:     "send to route error propagation",
+	//	TestFunc: testSendToRouteErrorPropagation,
+	//},
+	//{
+	//	Name:     "private channels",
+	//	TestFunc: testPrivateChannels,
+	//},
+	//{
+	//	Name:     "invoice routing hints",
+	//	TestFunc: testInvoiceRoutingHints,
+	//},
+	//{
+	//	Name:     "multi-hop payments over private channels",
+	//	TestFunc: testMultiHopOverPrivateChannels,
+	//},
+	//{
+	//	Name:     "query routes",
+	//	TestFunc: testQueryRoutes,
+	//},
+	//{
+	//	Name:     "route fee cutoff",
+	//	TestFunc: testRouteFeeCutoff,
+	//},
+	//{
+	//	Name:     "rpc middleware interceptor",
+	//	TestFunc: testRPCMiddlewareInterceptor,
+	//},
+	//{
+	//	Name:     "macaroon authentication",
+	//	TestFunc: testMacaroonAuthentication,
+	//},
+	//{
+	//	Name:     "bake macaroon",
+	//	TestFunc: testBakeMacaroon,
+	//},
+	//{
+	//	Name:     "delete macaroon id",
+	//	TestFunc: testDeleteMacaroonID,
+	//},
+	//{
+	//	Name:     "stateless init",
+	//	TestFunc: testStatelessInit,
+	//},
+	//{
+	//	Name:     "single hop invoice",
+	//	TestFunc: testSingleHopInvoice,
+	//},
+	//{
+	//	Name:     "wipe forwarding packages",
+	//	TestFunc: testWipeForwardingPackages,
+	//},
+	//{
+	//	Name:     "switch circuit persistence",
+	//	TestFunc: testSwitchCircuitPersistence,
+	//},
+	//{
+	//	Name:     "switch offline delivery",
+	//	TestFunc: testSwitchOfflineDelivery,
+	//},
+	//{
+	//	Name:     "switch offline delivery persistence",
+	//	TestFunc: testSwitchOfflineDeliveryPersistence,
+	//},
+	//{
+	//	Name:     "switch offline delivery outgoing offline",
+	//	TestFunc: testSwitchOfflineDeliveryOutgoingOffline,
+	//},
+	//{
+	//	Name:     "sendtoroute multi path payment",
+	//	TestFunc: testSendToRouteMultiPath,
+	//},
+	//{
+	//	Name:     "send multi path payment",
+	//	TestFunc: testSendMultiPathPayment,
+	//},
+	//{
+	//	Name:     "sendpayment amp invoice",
+	//	TestFunc: testSendPaymentAMPInvoice,
+	//},
+	//{
+	//	Name:     "sendpayment amp invoice repeat",
+	//	TestFunc: testSendPaymentAMPInvoiceRepeat,
+	//},
+	//{
+	//	Name:     "send payment amp",
+	//	TestFunc: testSendPaymentAMP,
+	//},
+	//{
+	//	Name:     "sendtoroute amp",
+	//	TestFunc: testSendToRouteAMP,
+	//},
+	//{
+	//	Name:     "forward interceptor dedup htlcs",
+	//	TestFunc: testForwardInterceptorDedupHtlc,
+	//},
+	//{
+	//	Name:     "forward interceptor",
+	//	TestFunc: testForwardInterceptorBasic,
+	//},
+	{
+		Name:     "forward interceptor modified htlc",
```

Comment:
You know you can run `make itest icase=forward_interceptor_modified_htlc` to select a single test, right? And in the IDE you can use this pattern: `^TestLightningNetworkDaemon/tranche.*/.*-of-.*/.*/forward_interceptor_modified_htlc`.
Just because that's probably more convenient than commenting out all other tests (in most cases).